pyproject.toml

```diff
@@ -0,0 +1,63 @@
+[build-system]
+requires = ["setuptools", "wheel", "setuptools_scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "graphix-symbolic"
+authors = [{ name = "Shinichi Sunami", email = "shinichi.sunami@gmail.com" }]
+maintainers = [
+    { name = "Shinichi Sunami", email = "shinichi.sunami@gmail.com" },
+]
+license = { file = "LICENSE" }
+description = "Symbolic plugin for graphix"
+readme = "README.md"
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Environment :: Console",
+    "Intended Audience :: Science/Research",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
```

Comment:
could add 3.12 to be compatible with `graphix` - or is 3.12 not supported by sympy?